pom.xml

```diff
@@ -297,6 +298,11 @@
                 <artifactId>licenses-generator</artifactId>
                 <version>1.0.5</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.lib</groupId>
+                <artifactId>lib-jenkins-maven-embedder</artifactId>
+                <version>3.15</version>
```

Comment:
What is the reason for including this please?

Reply:
`li-jenkins-maven-embedder` which is brought by  transitives of hudson-maven-embedder is of old version which creates runtime compatibility problem. `quarkus-bootstrap-resolver`  requires `3.15` version of `lib-jenkins-maven-embedder`

Reply:
Its been removed with new version of snow-drop-license-generator 
license-generator needs new version to be binary compatible with `quarkus-bootsrap-resolver`